paimon-flink/paimon-flink-common/src/main/java/org/apache/paimon/flink/action/cdc/mysql/MySqlActionUtils.java

```diff
@@ -202,4 +205,37 @@ static MySqlSource<String> buildMySqlSource(Configuration mySqlConfig) {
                 new JsonDebeziumDeserializationSchema(true, customConverterConfigs);
         return sourceBuilder.deserializer(schema).includeSchemaChanges(true).build();
     }
+
+    private static void validateMySqlConfig(Configuration mySqlConfig) {
+        checkArgument(
+                mySqlConfig.get(MySqlSourceOptions.HOSTNAME) != null
```

Comment:
`mySqlConfig.contains(MySqlSourceOptions.HOSTNAME)`
Just check `contains` is OK, if it is a empty string, detailed message will be reported in mysql client.

Reply:
remove the check whether it is empty